migrations/versions/6fc1a99cefd5_create_coastlines.py

```diff
@@ -41,23 +41,7 @@ def upgrade():
             nullable=False,
         ),
     )
-    # op.create_index(
-    #     "idx_coastlines_geom",
-    #     "coastlines",
-    #     ["geom"],
-    #     unique=False,
-    #     postgresql_using="gist",
-    #     postgresql_ops={},
-    # )
-    # op.create_index(op.f("ix_coastlines_id"), "coastlines", ["id"], unique=False)
 
 
 def downgrade():
-    # op.drop_index(
-    #     "idx_coastlines_geom",
-    #     table_name="coastlines",
-    #     postgresql_using="gist",
-    #     postgresql_ops={},
-    # )
-    # op.drop_index(op.f("ix_coastlines_id"), table_name="coastlines")
```

Comment:
Why was the docker file POSTing into the coastlines table in the first place?

Reply:
Would you please help me understand your question by explaining what you mean in greater detail?

Reply:
If you look at the commented out code, it looks like the code is POSTing into the DB with coastlines data. There is create_index and drop_index. Why is this code there in the first place? What purpose did it serve? It looked to me that was putting data into the DB.

Reply:
This file is responsible for updating the "shape" (i.e., schema) of our database. These are referred to as migrations, which is why the parent folder is named that. Google has a nice write up regarding database migrations [here](https://cloud.google.com/architecture/database-migration-concepts-principles-part-1#:~:text=Database%20migration%20is%20the%20process,restructured%2C%20in%20the%20target%20databases.), if you'd like to learn more about it.

The files in this folder are executed whenever you run `poetry run alembic upgrade head`.

> It looked to me that was putting data into the DB.

Our seed files/scripts are responsible for putting data into the DB (e.g., "server/seed_initial_data.py").

Reply:
Ahhh alright that clears things up.